Preface/Preface.pillar

```diff
@@ -58,12 +57,13 @@ available at *http://www.deepintopharo.org*
 
 Do not be frustrated by parts of Pharo that you do not immediately understand.
 You do not have to know everything! Alan Knight expresses this as follows:
-
+{{{latex:\begin{quote} }}}
```

Comment:
> When you prefix a line with @@XXX Pillar generates a LaTeX environment named XXX. It's up to the > book author to define this XXX LaTeX environment. This is typically in a file named support/latex/common.tex.

Ohh, I see! I thought it was hardcoded into Pillar itself. Ok, I'll fix the 'Notes' phrasing.
Is there a way to get these annotated tags to generate different HTML tags? Like, a way to specify that `@@quote` should generate an HTML `<blockquote>`?